src/components/ReportActionItem/MoneyRequestReportPreview/index.tsx

```diff
@@ -57,6 +57,11 @@ function MoneyRequestReportPreview({
         [StyleUtils, currentWidth, shouldUseNarrowLayout, transactions.length],
     );
 
+    const shouldShowIOUData = useMemo(() => {
+        const firstAttendee = transactions.at(0)?.comment?.attendees?.at(0);
+        return transactions.some((transaction) => firstAttendee?.email !== transaction.comment?.attendees?.at(0)?.email);
+    }, [transactions]);
```

Comment:
We dont want to do this for sure. 

Could you instead check if the report is IOU report AND go through the transaction, get the amount (modifiedAmount ?? amount) and only if there is positive AND some negative amount (that is request from the other user) we show the IOUData

lets use the transaction amounts then and if the signs flip, we know there are requests from both